src/components/Button/ProductTransferButton.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-btn
+    v-if="!item.sold && item.contentOf === 'No'"
+    variant="plain"
+    @click="openDialog"
+  >
+    <v-icon color="#607D8B" size="25">mdi-swap-horizontal</v-icon>
+  </v-btn>
+
+  <v-btn v-else variant="plain" disabled>
+    <v-icon color="#607D8B" size="25">mdi-swap-horizontal</v-icon>
+  </v-btn>
```

Comment:
You can get rid of the second button by:
```suggestion
  <v-btn
    :disabled="item.sold || item.contentOf === 'Yes'"
    variant="plain"
    @click="openDialog"
  >
    <v-icon color="#607D8B" size="25">mdi-swap-horizontal</v-icon>
  </v-btn>
```

Reply:
fixed